lib/src/components/Banner.tsx

```diff
@@ -67,72 +67,38 @@ function	DefaultVariant({variant, title, children, primaryButton, secondaryButto
 	)
 }
 
-function	ImageVariant({image, onClick}: {image: string, onClick?: React.MouseEventHandler}) {
+function	ImageVariant({image, height, onClick}: {height: string | number, image: string, onClick?: React.MouseEventHandler}) {
 	return (
 		<div className={'w-full h-full -ml-1 cursor-pointer'} style={{minWidth: 'calc(100% + 8px)'}} onClick={onClick}>
-			<img src={image} className={'relative object-cover w-full h-full'} loading={'eager'} />
+			<img src={image} height={height} className={'relative object-cover w-full'} loading={'eager'} />
```

Comment:
Maybe we should send the image as a ReactElement. This way you can put whatever you want, and we can use stuff like next/image with all the perf bonus. Not sure.

Reply:
ok! I'll do this